README.md

```diff
@@ -56,61 +57,97 @@ const converted = tabletojson.convert(html);
 console.log(converted);
 ```
 
-### Duplicate column headings 
+### Duplicate column headings
 
-If there are duplicate column headings, subsequent headings are suffixed with a count:
+If there are duplicate column headings, subsequent headings are suffixed with a
+count:
 
-```
-// Table
-| PLACE | VALUE | PLACE | VALUE |
-|   abc |     1 |   def |     2 |
+PLACE | VALUE | PLACE | VALUE
+------|-------|-------|------
+  abc |     1 |   def |     2
 
-// Example output
+```js
 [{
   PLACE: 'abc', VALUE: '1',
   PLACE_2: 'def', VALUE_2: '2',
 }]
 ```
 
-### Tables with rowspan 
-
-Having tables with rowspan, the content of the spawned cell must be available in the respective object.
+### Tables with rowspan
 
-```
-// Table
-| PARENT | CHILD | AGE |
-|        |   Tom |   3 |
-| Marry  | Steve |  12 |
-|        |   Sue |  15 |
+Having tables with rowspan, the content of the spawned cell must be available in
+the respective object.
 
-// Example output
+<table id="table11" class="table" border="1">
+    <thead>
+    <tr>
+        <th>Parent</th>
+        <th>Child</th>
+        <th>Age</th>
+    </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="3">Marry</td>
+            <td>Sue</td>
+            <td>15</td>
+        </tr>
+        <tr>
+            <td>Steve</td>
+            <td>12</td>
+        </tr>
+        <tr>
+            <td>Tom</td>
+            <td>3</td>
+        </tr>
+    </tbody>
+</table>
+
+```js
 [{
   PARENT: 'Marry', CHILD: 'Tom', AGE, '3',
   PARENT: 'Marry', CHILD: 'Steve', AGE, '12',
   PARENT: 'Marry', CHILD: 'Sue', AGE, '15'
 }]
 ```
 
-### Tables with complex rowspan 
+### Tables with complex rowspan
 
 Having tables with complex rowspans, the content of the spawned cell must be available in the respective object.
 
-```
-// Table
-| PARENT | CHILD | AGE |
-+--------+-------+-----+
-|        |   Sue |  15 |
-+        +-------+-----+
-| Marry  | Steve |  12 |
-+        +-------+-----+
-|        |       |     |
-+--------+   Tom |   3 +
-|        |       |     |
-+ Taylor +-------+-----+
-|        | Peter |  17 |
-+--------+-------+-----+
-
-// Example output
+<table id="table12" class="table" border="1">
```

Comment:
Same here .... Why HTML Code?

Reply:
IMHO it blows up the README...

Reply:
When reading it in raw yes, it looks ugly, but when rendered it show a real HTML table with rowspans, since markdown don't support them. There's a markdown extension to add rowspans support in tables, but github don't support it.